# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../e3/e3-make.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -up e3-2.8/Makefile.make e3-2.8/Makefile
--- e3-2.8/Makefile.make	2010-05-27 20:58:59.000000000 +0200
+++ e3-2.8/Makefile	2011-01-13 09:28:25.253712906 +0100
@@ -39,11 +39,11 @@ all:	32
 
 32:	$(ASOURCES) Makefile
 ifeq	($(OS),LINUX)
-	nasm -O2 -f bin -l e3.lst -o e3 e3.asm -DCRIPLED_ELF=1 -D$(OS) -D$(EXMODE) 
+	nasm -O2 -f bin -l e3.lst -o e3 e3.asm -DCRIPLED_ELF=1 -D$(OS)
 	chmod +x e3
 else
 	echo $(ASVER)
-	nasm -O2 $(AFLAGS) -o e3.o e3.asm -l e3.lst -D$(OS) -D$(EXMODE)
+	nasm -O2 $(AFLAGS) -o e3.o e3.asm -l e3.lst -D$(OS)
 	ld -s -o e3 e3.o
 	strip --remove-section .comment e3
 endif
@@ -71,13 +71,15 @@ endif
 # -------------------------------------------------
 
 install:	e3
-	install -d $(PREFIX) $(BINDIR) $(MANDIR)
+	install -d $(BINDIR) $(MANDIR)
 	install -m 755 e3 $(BINDIR)
-	ln -sf $(BINDIR)/e3 $(BINDIR)/e3ws
-	ln -sf $(BINDIR)/e3 $(BINDIR)/e3em
-	ln -sf $(BINDIR)/e3 $(BINDIR)/e3pi
-	ln -sf $(BINDIR)/e3 $(BINDIR)/e3vi
-	ln -sf $(BINDIR)/e3 $(BINDIR)/e3ne
+	(cd $(BINDIR); \
+		ln -sf $(BINDIR)/e3 $(BINDIR)/e3ws; \
+		ln -sf $(BINDIR)/e3 $(BINDIR)/e3em; \
+		ln -sf $(BINDIR)/e3 $(BINDIR)/e3pi; \
+		ln -sf $(BINDIR)/e3 $(BINDIR)/e3vi; \
+		ln -sf $(BINDIR)/e3 $(BINDIR)/e3ne; \
+	);
 	install -m 644 e3.man $(MANDIR)/e3.$(MANSEC)
 
 clean:
